RELEASE.md

```diff
@@ -208,14 +230,16 @@ The Buildkite release is fairly-well automated at this point, but once it is com
 ## The Builder Worker
 
 New `habitat/builder-worker` packages will automatically be built by
-Builder's `post_habitat_release` pipeline ([definition](https://github.com/habitat-sh/builder/blob/master/.expeditor/post_habitat_release.pipeline.yml)),
+Builder's `post_habitat_release` pipeline ([definition](https://github.com/habitat-sh/builder/blob/master/.expeditor/post_habitat_release.pipeline.yml),
 [Buildkite
 pipeline](https://buildkite.com/chef/habitat-sh-builder-master-post-habitat-release))
-when a Habitat release completes. Please follow along in that pipeline
-to ensure that the new packages are validated and properly promoted to
-the stable channel.
+when a Habitat release completes. The instructions below describe how to
+follow along in that pipeline to ensure that the new packages are validated and
+properly promoted to the stable channel.
+
+After the workers are built and promoted to acceptance by the pipeline, you will be prompted to "Evaluate habitat/builder-worker in Acceptance" in the pipeline. Use these [instructions](https://github.com/habitat-sh/builder/pull/1233) to connect to the `acceptance-worker` nodes. After connecting to the node, validate the version of the Supervisor and the version of `habitat/builder-worker`. An easy way to check the version is with `hab sup status`, the version should match the version output by the pipeline. Next, build a package in acceptance to ensure that the workers are behaving properly. This can be done through the [acceptance builder ui](https://bldr.acceptance.habitat.sh/). The key things you are looking for are that the builds succeed and that their build output indicates they are using the new stable versions of `hab`, `hab-plan-build` and `hab-studio`. You should follow these steps for both a Windows and a Linux worker.
 
```

Comment:
After thinking about this last week, I don't think it is necessary to ssh/rdp to the workers. I think one just needs to initiate a linux/windows/linux2 build and validate the versions of hab-studio, hab and hab-plan-build being used by looking at the build log. Brcause the workers take a dependency on hab-studio, if the new versions show up in the log, it would have to be a new worker. Because an older worker would have been built with a previous srudio that would have included the older hab and hab-plan-build.

As for confitming the version of the Supervisor, I think that is overkill here and is actually a new concept entirely we are introducing to the process. I know I was the one who mentioned we should check this during the last post-release steps but I think I suggested that out of ignorance.

I think if we could eliminate the need to remote to the workers that would make the release alot smoother. However I am going to add an issue to the automation epic suggesting we add worker and supervisor versions to the top of the build log.

Reply:
Agreed, that makes sense. Addressed in [28bbaa](https://github.com/habitat-sh/habitat/pull/7580/commits/28bbaaa60686aa86fee58624b60607751fe353a0)